bot/exts/moderation/infraction/_scheduler.py

```diff
@@ -352,9 +369,16 @@ async def deactivate_infraction(
             log_text["Failure"] = "The bot lacks permissions to do this (role hierarchy?)"
             log_content = mod_role.mention
         except discord.HTTPException as e:
-            log.exception(f"Failed to deactivate infraction #{id_} ({type_})")
-            log_text["Failure"] = f"HTTPException with status {e.status} and code {e.code}."
-            log_content = mod_role.mention
+            if e.code == 10007 or e.status == 404:
```

Comment:
What does error code `10007` imply?

Reply:
10007 is Unknown Member JSON error code.

Reply:
Thank you for this information.